src/Output/Output_DumpData_Total_HDF5.cpp

```diff
@@ -2352,7 +2353,9 @@ void FillIn_InputPara( InputPara_t &InputPara, const int NFieldStored, char Fiel
 #  ifdef MHD
    InputPara.Opt__InitBFieldByFile   = OPT__INIT_BFIELD_BYFILE;
 #  endif
-
+#  ifdef SUPPORT_FFTW
+   InputPara.Opt__FFTW_Startup       = OPT__FFTW_STARTUP;
```

Comment:
You need to call `H5Tinsert()` to insert this new parameter. Check `Opt__InitBFieldByFile` as an example.